cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -56,8 +56,8 @@ import org.ossreviewtoolkit.model.licenses.LicenseClassifications
 import org.ossreviewtoolkit.model.readValue
 import org.ossreviewtoolkit.notifier.Notifier
 import org.ossreviewtoolkit.reporter.HowToFixTextProvider
+import org.ossreviewtoolkit.reporter.Reporter
```

Comment:
commit-msg:  

I was a bit confused by the term split in "but it makes sense to split them".
Would it make sense to say "but it makes sense to move them into separate plugins"?

"Due to this dependency it is easier" it's not obvious to me why this dependency makes it harder, can you give a hint?

Reply:
I've clarified both things in a reworked commit message.